tests/e2e/models/generated_e2e_model_tests.cmake

```diff
@@ -0,0 +1,79 @@
+################################################################################
+# Autogenerated by build_tools/testing/generate_cmake_e2e_model_tests.py       #
+# To update the tests, modify definitions in the generator and regenerate this #
+# file.                                                                        #
+################################################################################
+
+set(IREE_MODULE_COMPILE_CONFIG_ID_ANDROID-ARM64-V8A
+  "1f2adf49-282e-4aff-9d4f-e63b1621f1e8"
+)
+
+set(IREE_MODULE_COMPILE_CONFIG_ID_RISCV32-LINUX
+  "6d9ce240-ec14-4d8f-a8e4-1b20aa17b4e4"
+)
+
+set(IREE_MODULE_COMPILE_CONFIG_ID_RISCV64-LINUX
+  "cdf579a9-5446-403b-a991-802a6c702e65"
+)
+
+set(IREE_MODULE_COMPILE_CONFIG_ID_X86_64
+  "e7e18b0f-c72d-4f1c-89b1-5afee70df6e9"
+)
+
+iree_benchmark_suite_module_test(
+  NAME
+    "mobilenet_v1_fp32_correctness_test"
+  MODEL
+    "78eab9e5-9ff1-4769-9b55-933c81cc9a0f_MobileNetV1_fp32"
+  DRIVER
+    "local-sync"
+  EXPECTED_OUTPUT
+    "mobilenet_v1_fp32_expected_output.txt"
+  RUNNER_ARGS
+    "--entry_function=main"
+    "--function_input=1x224x224x3xf32=0"
+)
+
+iree_benchmark_suite_module_test(
+  NAME
+    "efficientnet_int8_correctness_test"
+  MODEL
+    "4a6f545e-1b4e-41a5-9236-792aa578184b_EfficientNet_int8"
+  DRIVER
+    "local-sync"
+  EXPECTED_OUTPUT
+    "efficientnet_int8_expected_output.txt"
+  RUNNER_ARGS
+    "--entry_function=main"
+    "--function_input=1x224x224x3xui8=0"
+)
+
+iree_benchmark_suite_module_test(
+  NAME
+    "deeplab_v3_fp32_correctness_test"
```

Comment:
https://github.com/iree-org/iree/issues/10748. This may not work.

Reply:
Looks like #10748 has been resolved (https://github.com/iree-org/iree/commit/cc183199c11df2ce3cd8696251ca2c30210b36e9)